### ISSUE TITLE
Problem at the beginning of use

### ISSUE THREAD

**Issue by Matteo945** [christoph2/pyxcp#177]
Is the library supported on 64-bit Python on Windows currently?

**Comment by christoph2** [comment]
Yes, of cause; pyXCP runs basically on Windows, Linux, and MacOS.
There are even pre-build Windows packages available, so you may start with
`pip install pyxcp`
I'm still working on reasonable documentation, for now you may find some useful tips [here](https://github.com/christoph2/pyxcp/discussions/165).
